utils/semodule-utils/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=semodule-utils
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/20200710
+PKG_HASH:=0cc37f9cec751d9c2abb5f2b228b060567e973cb47c19b53b8a4a7378baaa853
+PKG_INSTALL:=1
+HOST_BUILD_DEPENDS:=libsemanage/host
+
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/semodule-utils
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:= +policycoreutils
+  TITLE:=SELinux policy utilities for dealing with modules
+  URL:=http://selinuxproject.org/page/Main_Page
+endef
+
+define Package/semodule-utils/description
+	Semodule-utils is a collection of utilities for dealing with
+	policy modules.
+endef
+
+include $(INCLUDE_DIR)/host-build.mk
```

Comment:
Move that up to the other `include` as well please.

Reply:
Fixed.